src/index.ts

```diff
@@ -80,6 +87,7 @@ type ParsedUrlQueryInput = { [key: string]: unknown };
 const serializeImgixUrlQueryParamValues = (query: ImgixUrlQueryParams): ParsedUrlQueryInput =>
     pipe(
         (): Record<keyof ImgixUrlQueryParams, string | number | undefined> => ({
+            ar: typeof query.ar !== 'undefined' ? `${query.ar.w}:${query.ar.h}` : undefined,
```

Comment:
I think 
I think you can use `mapValueIfDefined` here, e.g.

```ts
ar: mapValueIfDefined(ar => `${ar.w}:${ar.h}`)
```

Reply:
Ah yep, that worked.

Reply:
I have to mention while I'm here that I found the original plain vanilla JS approach more legible, easier to comprehend. :v: